cumulusci/tasks/salesforce/Deploy.py

```diff
@@ -69,7 +69,8 @@ def _init_options(self, kwargs):
 
     def _get_api(self, path=None):
         if not path:
-            path = self.task_config.options__path
+            path = self.options.get("path")
+            assert path, f"Path should be specified for {self.__class__.name}"
```

Comment:
I'll go ahead an approve since this is a nitpick about an edge case that we don't think should happen, but the task config name (`self.name`) would probably be more helpful here.

Reply:
In my 2 minutes of testing, self.name was None. Perhaps because it was run in the context of a unit test.

```
(Pdb) self.name
(Pdb) self.__class__.name
'BaseSalesforceMetadataApiTask'
```